org.adempiere.base/src/org/compiere/model/X_AD_Reference.java

```diff
@@ -252,29 +256,27 @@ public KeyNamePair getKeyNamePair()
         return new KeyNamePair(get_ID(), getName());
     }
 
-	/** ValidationType AD_Reference_ID=2 */
-	public static final int VALIDATIONTYPE_AD_Reference_ID=2;
-	/** DataType = D */
-	public static final String VALIDATIONTYPE_DataType = "D";
-	/** List Validation = L */
-	public static final String VALIDATIONTYPE_ListValidation = "L";
-	/** Table Validation = T */
-	public static final String VALIDATIONTYPE_TableValidation = "T";
-	/** Set Validation type.
-		@param ValidationType Different method of validating data
+	/** ShowInactive AD_Reference_ID=200230 */
+	public static final int SHOWINACTIVE_AD_Reference_ID=200230;
+	/** No = N */
+	public static final String SHOWINACTIVE_No = "N";
+	/** Yes = Y */
+	public static final String SHOWINACTIVE_Yes = "Y";
```

Comment:
and the option for "No With Context Menu" ?

Reply:
Hi @hengsin, @norbertbede had a chat with @nmicoud - according to that, the implementation can be either simple or advanced (due to client, org, role, user-wide options). We think the idea is great so we decided to create a new ticket and take the scope of this improvement there. It could be implemented after the design is approved.

Reply:
@hengsin  the new issue created https://idempiere.atlassian.net/browse/IDEMPIERE-5577